Week02/lesson04/homework/relaxr_starter_code/index.html

```diff
@@ -1,27 +1,83 @@
 <!DOCTYPE html>
+
 <html lang="en">
+
 <head>
+
 	<meta charset="utf-8">
 	<meta name="description" content="Your description goes here">
 	<meta name="keywords" content="one, two, three">
-	<title>Blank Template</title>
+	<title>Relaxr</title>
 
-	<!-- external CSS links -->
 	<link rel="stylesheet" href="css/reset.css">
 	<link rel="stylesheet" href="css/style.css">
+
 </head>
 
 <body>
 
-	<div id="container">
-		<header>
-			<h1>Blank Template</h1>
-			<h2>Just another blank template</h2>
-		</header>
+<div id="container">
+
+<header>
+
+			<h1>Relaxr</h1>
+
+			<h2>Get piece of mind with a single tap</h2>
+
+				<nav>
```

Comment:
Single button probably isn't enough "interaction" to be considered a nav.

I'd use the `<a>` like you did and then use `padding` and `background-color` `text-align: center`and `border-radius` to achieve the desired effect directly on the `<a>`, it doesn't really need a wrapper here